components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -144,6 +217,13 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:create";
     };
+  /*
+  Create a new role
+
+  Creates a new role to be used in the IAM policies that control permissions in Automate.
+
+  A role defines the scope for a set of actions.
```

Comment:
s/defines the scope for a set of actions/defines the scope of actions in a policy statement